crates/pathfinder/src/storage/merkle_tree.rs

```diff
@@ -479,14 +488,14 @@ mod tests {
             });
 
```

Comment:
For this and other test cases could we somehow find an interesting enough persistent subtree somehow because the tests are no longer interesting for leaves.

Reply:
Discussed this, we can let this wait until we are looking at deletion functinality, it's difficult to get a useful tree here without counting on leafs.